src/components/filters/FristFilter.tsx

```diff
@@ -0,0 +1,32 @@
+import { CheckboxGroup, Checkbox } from '@navikt/ds-react';
+import { FristFilterOption } from '@/utils/hendelseFilteringUtils';
+import React from 'react';
+
+const texts = {
+  legend: 'Velg synlighet for frist',
+  option: {
+    past: 'Tidligere datoer',
```

Comment:
"før dagens dato" høres mer logisk ut i mitt hode, men veilederne skjønner sikkert dette også. Vi får bare prøve ut :smile:

Reply:
Kan godt gå for "Før dagens dato" 🙌